exercises/security/README.md

```diff
@@ -256,6 +256,10 @@ You can test by using a stored procedure, so the application must be sanitize th
 
 <details>
 <summary>Explain CVE and CVSS</summary><br><b>
+ CVE stands for Common Vulnerabilities and Exposures, it makes easier to identify vulnerabilities by adding an ID written on 8 digits.
```

Comment:
You are right that there is an ID but, what I think is missing is why exactly it makes it easier to identify vulnerabilities.  I would add an explanation on who issues CVEs and perhaps something about how this system works. Without that, it's really hard to grasp the idea of what a CVE is and how it's helpful to identify vulnerabilities